riot.im.nightly/config.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Why `riot.im.nightly` as a single directory...?

It seems like it might be more natural to have something like

* riot.im
  * release
    * config.json
  * nightly
    * config.json

Reply:
yeah, the probably would make more sense